src/components/molecules/Visualizer/Infobox/index.tsx

```diff
@@ -89,68 +89,86 @@ const Infobox: React.FC<Props> = ({
       onEnter={() => setIsReadyToRender(false)}
       onEntered={() => setIsReadyToRender(true)}
       onExit={() => setIsReadyToRender(false)}>
-      {blocks?.map((b, i) => (
-        <Field
-          key={b.id}
-          id={b.id}
-          index={i}
-          isEditable={isEditable}
-          isBuilt={isBuilt}
-          isSelected={selectedBlockId === b.id}
-          dragDisabled={blocks.length < 2}
-          renderInsertionPopUp={
-            isReadyToRender &&
-            insertionPopUpPosition?.[0] === i &&
-            renderInsertionPopUp?.(handleBlockInsert, onInsertionPopUpClose)
-          }
-          insertionPopUpPosition={insertionPopUpPosition?.[1]}
-          onMove={onBlockMove}
-          onInsert={p => onInsertionButtonClick?.(i, p)}>
-          <PluginBlock
-            block={b}
-            isSelected={!!isEditable && !isBuilt && selectedBlockId === b.id}
+      <Wrapper
+        paddingTop={property?.default?.infoboxPaddingTop}
+        paddingBottom={property?.default?.infoboxPaddingBottom}
+        paddingLeft={property?.default?.infoboxPaddingLeft}
+        paddingRight={property?.default?.infoboxPaddingRight}>
```

Comment:
I know it is more difficult, but I think instead of adding a wrapper here, we should be passing the padding into the Frame and updating the Frame component to avoid unnecessary components.